pkg/analysis/passes/gomanifest/gomanifest.go

```diff
@@ -167,43 +176,23 @@ func verifyManifest(manifest map[string]string, goFiles []string, sourceCodeDir
 	return nil
 }
 
-func hashFileContent(path string) (string, error) {
-	// Handle hashing big files.
-	// Source: https://stackoverflow.com/q/60328216/1722542
-
-	f, err := os.Open(path)
+// we hash the file content using the sha256 algorithm
+// a second hash is created using the line endings that windows uses
+// for plugins compiled with native windows that use windows line endings
+func hashFileContent(path string) (string, string, error) {
+	data, err := os.ReadFile(path)
 	if err != nil {
-		return "", err
+		return "", "", err
 	}
 
-	defer func() {
-		err = f.Close()
-		if err != nil {
-			fmt.Printf("error closing file for hashing: %v", err)
-		}
-	}()
+	// Calculate the original hash
+	hOrig := sha256.Sum256(data)
+	originalHash := hex.EncodeToString(hOrig[:])
```

Comment:
Did you consider normalize here as well similar to how you do source map comparison?

Reply:
the go manifest is a list of the hashed content of the go files at build, the hash is calculated using the line ending of the OS that wrote and we do here is rehash the file and verify. Here we are basically detecting which line ending was used by calculating both hashes and checking.